### PR TITLE
Fixed gate Portcullis

### PR DESCRIPTION
**Pull request by grimgravy**
Fixed gate Portcullis. When the chiefs approached to attack automatically the door shut.

**Changes proposed:**

-  
-  
-  

**Target branch(es):** 1.x/2.x etc.

**Issues addressed:** Closes #

**Tests performed:** (Does it build, tested in-game, etc)

**Known issues and TODO list:**

- [ ] 
- [ ] 

**NOTE** You no longer need to squash your commits, on merge we will squash it for you. (GitHub added a new feature)

Add the trial_of_the_champion.h file

DATA_PORTCULLIS,

GO_PORTCULLIS = 195650,

### CODE REVIEW

**Comment by sucofog** [comment]
Whats this? lol

File: src/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/instance_trial_of_the_champion.cpp [hunk 1]
```diff
@@ -191,6 +193,9 @@ class instance_trial_of_the_champion : public InstanceMapScript
                     GO_MainGateGUID = go->GetGUID();
                     HandleGameObject(GO_MainGateGUID, false, go);
                     break;
+                case GO_PORTCULLIS
```
Comment (sucofog) [inline src/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/instance_trial_of_the_champion.cpp]: case GO_PORTCULLIS:

File: src/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/instance_trial_of_the_champion.cpp [hunk 2]
```diff
@@ -191,6 +193,9 @@ class instance_trial_of_the_champion : public InstanceMapScript
...
+					GO_PortcullisGUID = go-GetGUID();
```
Comment (sucofog) [inline src/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/instance_trial_of_the_champion.cpp]: GO_PortcullisGUID = go->GetGUID();

File: src/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/instance_trial_of_the_champion.cpp [hunk 3]
```diff
@@ -191,6 +193,9 @@ class instance_trial_of_the_champion : public InstanceMapScript
...
+					break;
```
Comment (sucofog) [inline src/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/instance_trial_of_the_champion.cpp]: -- before
HandleGameObject(go->GetGUID(), false, go);

File: src/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/instance_trial_of_the_champion.cpp [hunk 4]
```diff
@@ -446,6 +451,8 @@ class instance_trial_of_the_champion : public InstanceMapScript
                     return NPC_AnnouncerGUID;
                 case DATA_PALETRESS:
                     return NPC_ArgentChampionGUID;
+                case DATA_PORTCULLIS
```
Comment (sucofog) [inline src/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/instance_trial_of_the_champion.cpp]: case DATA_PORTCULLIS:

**Comment by sucofog** [comment]
Why you don't added the header data?

**Comment by grimgravy** [comment]
Because the header I added was for patch-3. Gave this problem then I could not ruin

**Comment by Yehonal** [comment]
@grimgravy would be possible to fix the compile errors so we can test it?

**Comment by Yehonal** [comment]
i've fixed compilation, now should be tested ;)

**Comment by ShinDarth** [comment]
remember to squash the commits when merging

**Comment by Yehonal** [comment]
imported on staging-branch, available on demo server

**Comment by Yehonal** [comment]
I've noticed that it also should fix this https://wowgaming.altervista.org/aowow/?achievement=3804 (to be tested)

**Comment by Jagermaister95** [comment]
The achievement works but the door opens and closes even after starting and killed the chiefs.

**Comment by grimgravy** [comment]
I apologize when applying a patch, because I do not know how to git right git so I gave this problem

**Comment by Yehonal** [comment]
Since tests are not satisfying and the author seems inactive, i've to close this PR

**Comment by Yehonal** [comment]
Alan thanks for reply, but even if i've fixed the compilation error ...we
didn't noticed any changes to the portcullis.. could you help us to
understand what should happen? Maybe with a video or screenshots. And what
your PR does. In this way we can reopen it maybe

Il 14 Set 2017 17:15, "Alan" <notifications@github.com> ha scritto:

**Comment by grimgravy** [comment]
As promised, I noticed the
Portcullis


[0001-Implemented-of-Portcullis.zip](https://github.com/azerothcore/azerothcore-wotlk/files/1424221/0001-Implemented-of-Portcullis.zip)

https://youtu.be/2s8JZH9ELUw

**Pull request closed without merging**